polaris-for-figma/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "api": "1.0.0",
+  "name": "Polaris for Figma",
+  "main": "build/code.js",
+  "ui": "build/ui.html",
```

Comment:
Same here, and for the `main` key. Can we use `dist/us.html` and `dist/code.js` respectively?

Also for consistency, is it weird to use `index.js` and `index.html`? Not as descriptive but also follows convention